### ISSUE TITLE
wsl build error

### ISSUE THREAD

**Issue by xiusin** [vlang/v#487]
```
xiusin@DESKTOP-M945IRT:~/code/v/compiler$ cc -std=c11 -w -o vc v.c
xiusin@DESKTOP-M945IRT:~/code/v/compiler$ ./vc -o v .

Segmentation fault (core dumped)
```

**Comment by xiusin** [comment]
ubuntu 19.04

**Comment by whoizit** [comment]
#488

**Comment by Oxylibrium** [comment]
Should be fixed with v0.1.0
